cache/disk/lru.go

```diff
@@ -4,6 +4,26 @@ import (
 	"container/list"
 	"errors"
 	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	gaugeCacheSizeBytes = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "bazel_remote_disk_cache_size_bytes",
+		Help: "The current number of bytes in the disk backend",
+	})
```

Comment:
We expect the cache size to increase until it reaches the limit, and then hover around there. What else would we look out for in this graph?

Reply:
Yes. In some deployments it can take several weeks to reach the limit, and during that time it shows how fast the cache is growing and gives early indication about if the limit is high enough to keep entries for a reasonable amount of time. And also if growing with constant rate or flattern out after all products have been built at least once. Or if there are sudden jumps, either expected or unexpected, that could indicate if artifacts are not shared as desired.

After limit has been reached, it becomes a convenient way to show current configured limit for several caches, on same dashboard that shows other cache metrics. Observing that it hovers around the limit makes me feel that bazel-remote is working correctly. ;-)

I had this patch for a long time, and my users seems to have an intuitive understanding about what it means.

Reply:
On a dashboard I show this value in a graph, with date on x axis, not only what the value is right now.

Reply:
Sounds reasonable.